src/ResourceManager/TrafficManager/Commands.TrafficManager2/ChangeLog.md

```diff
@@ -18,13 +18,17 @@
         - Additional information about change #1
 -->
 ## Current Release
-* Fixed issue with default resource groups not being set.
-
-## Version 4.0.9
 * Updated to the latest version of the Azure ClientRuntime.
 
 ## Version 4.0.8
-* Updated help files to include full parameter types and correct input/output types.
```

Comment:
I'm not sure what happened here, please revert all changes and just add your message under "Current Release"

Reply:
Making my best guess here; hope this is correct. I thought version 4.0.9 was the current release, didn't think both tags were supposed to be present.